### PR TITLE
en,zh: add descrption about cluster domain

### LINKED ISSUE #1241: Add more description for the cluster domain in the across Kubernetes deployment doc
## Change Request

**This repository is ONLY used to solve issues related to TiDB Operator documentation.
For other issues (related to TiDB, PD, etc), please move to [other repositories](https://github.com/pingcap/).**

Please answer the following questions before submitting your issue. Thanks!

1. Describe what you find is inappropriate or missing in the existing docs.
We request the user to configure the cluster domain for the TidbClusters in the different Kubernetes clusters, however, we should make it clear that the cluster domain configured for the TC should match with that of the Kubernetes cluster.

2. Describe your suggestion or addition.

3. Provide some reference materials (documents, websites, etc) if you could.

### PR DESCRIPTION
**Pull request by KanShiori**
### What is changed, added, or deleted? (Required)
Close #1241 

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/deploy-tidb-cluster-across-multiple-kubernetes.md [hunk 1]
```diff
@@ -84,7 +84,10 @@ spec:
 EOF
 ```
 
-上述配置中，字段 `spec.acrossK8s: true` 表示是跨 Kubernetes 集群部署 TiDB 集群，必须设置。
+相关字段含义如下：
+
+* `spec.acrossK8s`：表示是跨 Kubernetes 集群部署 TiDB 集群，必须设置为 `true`。
+* `spec.clusterDomain`：设置后，会使用包含 Cluster Domain 的 Pod FQDN 作为组件间相互访问的地址。如果 Pod 访问其他 Kubernetes 集群的 Pod FQDN 时需要 Cluster Domain，那么必须设置。
```
Comment (DanielZhangQD) [inline zh/deploy-tidb-cluster-across-multiple-kubernetes.md:90]: 这里拿 pd-0 举一个具体的例子吧

File: zh/deploy-tidb-cluster-across-multiple-kubernetes.md [hunk 2]
```diff
@@ -84,7 +84,14 @@ spec:
 EOF
 ```
 
-上述配置中，字段 `spec.acrossK8s: true` 表示是跨 Kubernetes 集群部署 TiDB 集群，必须设置。
+相关字段含义如下：
+
+* `spec.acrossK8s`：表示是跨 Kubernetes 集群部署 TiDB 集群，必须设置为 `true`。
```
Comment (ran-huang) [inline zh/deploy-tidb-cluster-across-multiple-kubernetes.md:89]: How about this?
```suggestion
* `spec.acrossK8s`：表示该 TiDB 集群是否要跨 Kubernetes 集群部署，本例中必须设置为 `true`。
```

File: zh/deploy-tidb-cluster-across-multiple-kubernetes.md [hunk 3]
```diff
@@ -84,7 +84,14 @@ spec:
...
+* `spec.clusterDomain`：设置后，会使用包含 Cluster Domain 的 Pod FQDN 作为组件间相互访问的地址。
+
+    以 Pod `${tc_name}-pd-0` 举例，其他 Kubernetes 集群的 Pod 会使用地址 `${tc_name}-pd-0.${tc_name}-pd-peer.${ns}.svc.${cluster_domain}` 来访问该 Pod。
+
+    如果 Pod 访问其他 Kubernetes 集群的 Pod FQDN 时需要 Cluster Domain，那么必须设置。
```
Comment (ran-huang) [inline zh/deploy-tidb-cluster-across-multiple-kubernetes.md:94]:
```suggestion
    如果 Pod 访问其他 Kubernetes 集群的 Pod FQDN 时需要 Cluster Domain，那么必须设置该字段。
```

**Review by ran-huang — APPROVED** [review]
LGTM

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Pull request merged**